DuckDuckGo/Statistics/GeneralPixel.swift

```diff
@@ -1297,6 +1297,8 @@ enum GeneralPixel: PixelKitEventV2 {
                 parameters[NewTabSearchBoxExperimentPixel.Parameters.onboardingCohort] = onboardingCohort.rawValue
             }
             return parameters
+        case .fileDownloadCreatePresentersFailed(let OSVersion):
+            return ["OSVersion": OSVersion]
```

Comment:
Can we call this "osVersion"? Some of other parameters are like this:
```swift
        public static let appVersion = "appVersion"
        public static let pixelSource = "pixelSource"
        public static let osMajorVersion = "osMajorVersion"
```

Also the associated parameter of the enum value should be `osVersion`, per convention.

Reply:
Yes, sure!

Reply:
Done ✅